test/signalrclient-e2e-tests/hub_connection_tests.cpp

```diff
@@ -0,0 +1,227 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#pragma once
+
+#include "stdafx.h"
+#include <string>
+#include "cpprest\details\basic_types.h"
+#include "cpprest\json.h" 
+#include "connection.h"
+#include "hub_connection.h"
+#include "url.h"
+
+TEST(hub_connection_tests, connection_status_start_stop_start_reconnect)
+{
+    auto hubConn = std::make_shared<signalr::hub_connection>(url + U("SignalR"));
```

Comment:
Fix casing - don't use pascal case since we don't use it in the entire code base.